utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -237,6 +273,46 @@ inline void set_descriptor_layout( RpptDescPtr srcDescPtr, RpptDescPtr dstDescPt
     }
 }
 
+// sets roi xywh values and dstImg sizes
+inline void set_roi(vector<string>::const_iterator imagePathsStart, vector<string>::const_iterator imagePathsEnd, RpptROI *roiTensorPtrSrc, RpptROI *roiTensorPtrDst, RpptImagePatchPtr dstImgSizes)
+{
+    tjhandle tjInstance = tjInitDecompress();
+    int i = 0;
+    for (auto imagePathIter = imagePathsStart; imagePathIter != imagePathsEnd; ++imagePathIter)
+    {
+        const string& imagePath = *imagePathIter;
+        FILE* jpegFile = fopen(imagePath.c_str(), "rb");
+        if (!jpegFile) {
+            std::cerr << "Error opening file: " << imagePath << std::endl;
+            continue;
+        }
+
+        fseek(jpegFile, 0, SEEK_END);
+        long fileSize = ftell(jpegFile);
+        fseek(jpegFile, 0, SEEK_SET);
+
+        std::vector<unsigned char> jpegBuffer(fileSize);
+        fread(jpegBuffer.data(), 1, fileSize, jpegFile);
```

Comment:
Are we reading the entire image data to this jpegBuffer?
I assume this fileSize is the total image size size (640x480x3)

if this is correct, how this is different to decoding the image?

Reply:
This code is for reading and decompressing a JPEG file and to get the image dimensions and subsampling. It does not perform the actual decoding of the image pixel data
Decoding the image pixel data involves further processes , such as allocating memory for the image data, calling the decompression functions to get the pixel values
fileSize will be the size of jpeg file

Reply:
Okay
What is the fileSize value let's say for a 640x480 image?

Reply:
okay as long as we aren't decoding the whole image twice like before.